test/functionalTest/cases/1038_ngsiv2_geo_location_entities/entity_location_shape_changes.test

```diff
@@ -0,0 +1,539 @@
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,5;9,9' --json
+echo
+echo
+
+
+echo "07. Entity changes to box"
+echo "=========================="
+payload='{
+  "value": [ "2, 2", "7, 7" ],
+  "type": "geo:box"
+}'
+orionCurl -X PUT --url /v2/entities/A/attrs/location --payload "$payload" --json
+echo
+echo
+
+
+echo "08. B1: ok, B2: ok, B3: ok, B4: ok"
+echo "=================================="
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,0;4,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,5;4,9' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,0;9,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,5;9,9' --json
+echo
+echo
+
+
+echo "09. Entity changes to triangle"
+echo "=============================="
+payload='{
+  "value": [ "2, 2", "7, 7", "7, 2", "2, 2" ],
+  "type": "geo:polygon"
+}'
+orionCurl -X PUT --url /v2/entities/A/attrs/location --payload "$payload" --json
+echo
+echo
+
+
+echo "10. B1: ok, B2: nok, B3: ok, B4: ok"
+echo "==================================="
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,0;4,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,5;4,9' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,0;9,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,5;9,9' --json
+echo
+echo
+
+
+echo "11. Entity changes to none"
+echo "=========================="
+payload='{
+  "value": "foo",
+  "type": "no-location"
+}'
+orionCurl -X PUT --url /v2/entities/A/attrs/location --payload "$payload" --json
+echo
+echo
+
+
+echo "12. B1: nok, B2: nok, B3: nok, B4: nok"
+echo "======================================"
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,0;4,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=0,5;4,9' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,0;9,4' --json
+orionCurl --url '/v2/entities?georel=intersects&geometry=box&coords=5,5;9,9' --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Create entity, starting as no-location
+==========================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/A?type=T
+Date: REGEX(.*)
+
+
+
+02. B1: nok, B2: nok, B3: nok, B4: nok
+======================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+
+
+03. Entity changes to point
+===========================
+HTTP/1.1 204 No Content
+Content-Length: 0
+Date: REGEX(.*)
+
+
+
+04. B1: ok, B2: nok, B3: nok, B4: nok
```

Comment:
It would be great to have someting more "compact" for this step output, instead the HTTP response for each case. Something like:

```
04. B1: ok, B2: nok, B3: nok, B4: nok
=====================================
B1: ok
B2: nok
B3: nok
B4: nok
```

After some time (one hour or so) trying to get that with grep, awk and similar bash black magic, I gave up... If you want to improve, please go ahead :)